_GuestTutorials/009-jabril-color-predictor.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Guest Tutorial #9: Color Predictor with Jabril"
+video_number: 9
+date: 2018-04-10
+video_id: iN3WAko2rL8
+repository: https://github.com/Jabrils/Color-Predictor-Project
+
+custom_sections:
+
+  - title: "Contact Jabril"
+    items:
+      - title: "Twitter"
+        url: "https://twitter.com/sefdstuff"
+      - title: "Instagram"
+        url: "https://www.instagram.com/sefdstuff"
+      - title: "Github"
+        url: "https://www.github.com/SEFDStuff"
+---
+
+Jabril visits the Coding Train to present a new project: A Machine Learning Color Predictor!
+
+Self-proclaimed "full time candy consumer", Jabril explores the realm of software development, employing his unique skill of speaking without moving his lips. We are highly entertained as he dabbles through AI, machine learning, robotics and video games.
```

Comment:
Did you write this description? I love it!

Reply:
I did! Thank you! mikakruschel fixed the compile problem in #1832, but I understand now. Thanks for all the great content. Keep it up!

Reply:
Lovely! Accepting suggestions for new descriptions for any and all videos! Anything that changes I should make sure it syncs up with the YouTube description (cc @cdslls)